### ISSUE TITLE
Floating Display Widget

### ISSUE THREAD

**Issue by psychemedia** [takluyver/mobilechelonian#17]
At the moment, the display canvas seems to be anchored as the output of the code cell that creates it.

It would be useful if the canvas could float / be dragged around the notebook so that as a script is built up over several cells, each step can be inspected in turn without having to try to scroll back to the display canvas to see what's happening.

There is an example of a floating canvas in this (old) notebook turtle repo: https://github.com/gkvoelkl/ipython-turtle-widget

**Comment by choldgraf** [comment]
I think this could be made fairly straightforward if JupyterLab were supported (if I recall, you can use a single window for the outputs of cells)

**Comment by psychemedia** [comment]
@choldgraf Yes, it struck me that if it worked in JupyterLab at all, then the canvas should be tear offable into its own window? But in a notebook UI, a floating panel would be easier. (I suspect that notebooks are far friendlier a UI than JupyterLab for novices, who are the likely target for `turtle`...)

**Comment by choldgraf** [comment]
@psychemedia for sure - we still use notebooks for teaching for the same reason...I'm hoping that Yuvi's simplest notebook setup could help with this https://github.com/yuvipanda/simplest-notebook
